### PR TITLE
IAM Docs: Describe the setup for a multitenant configuration with AzureAD

### PR DESCRIPTION
**Pull request by linoman**
**What is this feature?**

This add a description on how to add multiple Azure AD idps to Grafan through Keycloak.

**Why do we need this feature?**

This will allows users to setup multiple Azure tenants under the same Grafana instance. As referenced here: https://github.com/grafana/identity-access-team/issues/689

**Who is this feature for?**

IAM

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by volcanonoodle — APPROVED** [review]
Looks great! 🚀 

I noticed on other docs, like  [Configure Azure AD OAuth2 authentication](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/azuread/#configure-azure-ad-oauth2-authentication), we use bold text to highlight menu options and the like when giving instructions e.g. _Under **Manage** in the side menu, click **App Registrations > New Registration**. Enter a descriptive name._

I think it helps readability, but it is extra work and it might not be worth it. Just wanted to mention it!

File: docs/sources/setup-grafana/configure-security/configure-authentication/keycloak-multitenant/index.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+weight: 1350
+---
+
+# Multiple providers with Keycloak in Grafana
+
+While Grafana offers a variety of authentication providers, you can only configure one provider of one type at a time. However, you can configure multiple providers of the same type with the help of Keycloak.
+
+This guide explains how to set up multiple providers of the same type with Keycloak as an authentication provider in Grafana.
+
+The idea is to setup multiple OIDC providers in Keycloak with different tenants and configure Grafana to use the same Keycloak instance as the authentication provider.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-authentication/keycloak-multitenant/index.md:27]: setup should be "set up".

File: docs/sources/setup-grafana/configure-security/configure-authentication/keycloak-multitenant/index.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+1. Select **OpenID Connect v1.0**.
+   1. Select a unique **Alias** and **Display name**.
+   1. Copy the **Redirect URI**.
+   1. Back in Azure Portal, go to the application's **Authentication** section.
+   1. Add a **new platform** and select **Web**.
+   1. Paste the **Redirect URI** from Keycloak.
+   1. Save the changes.
+   1. Head over to the Azure Application overview and look for the **Endpoints** tab.
+   1. Copy the **OpenID Connect metadata document** URL.
+   1. Head back to Keycloak and paste the URL in the **Discovery endpoint** field.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-authentication/keycloak-multitenant/index.md:79]: same here - use the word "Navigate"

File: docs/sources/setup-grafana/configure-security/configure-authentication/keycloak-multitenant/index.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+   1. Head over to the Azure application overview and look for the **Application (client) ID**.
+   1. Copy the **Application ID** and paste it in the **Client ID** field in Keycloak.
+   1. Paste the client secret we created in the previous step in the **Client secret** field.
+   1. Click Add.
+
+{{% admonition type="note" %}}
+Up to this point, we have created an App Registration in Azure AD, assigned users to the application, created credentials for the application, and configured the application in Keycloak. In the Keycloak Client's section, the client with ID `account` Home URL can be used to test the configuration. This will open a new tab where we can login into the correct Keycloak realm with the Azure AD tenant we just configured.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-authentication/keycloak-multitenant/index.md:86]: remove "we" and use "you"

File: docs/sources/setup-grafana/configure-security/configure-authentication/keycloak-multitenant/index.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+{{% /admonition %}}
+
+Repeat this steps, for every Azure AD tenant you want to configure in Keycloak.
+
+#### Configure Grafana to use Keycloak
+
+Now that the Azure AD tenants are configured in Keycloak, we can configure Grafana to use Keycloak as the authentication provider.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-authentication/keycloak-multitenant/index.md:93]: remove "we" and use "you"

**Comment by linoman** [comment]
I've addressed all of the docs from the Docs squad in a follow-up PR https://github.com/grafana/grafana/pull/88134

**Pull request merged**